action/protocol/execution/evm/evm.go

```diff
@@ -200,8 +190,8 @@ func newParams(
 			GasPrice: execution.GasPrice(),
 		},
 		execution.Nonce(),
-		execution.Amount(),
```

Comment:
why change to `Value()`

Reply:
for EVM part, to make our code as close to ether as possible